frontend/src/tests/services/courseSearches.test.js

```diff
@@ -0,0 +1,131 @@
+import { fetchBasicCourseJSON, fetchBasicCourseHistoryJSON, fetchCourseHistoryNameQtrJSON, fetchGeQtrJSON, fetchInstructorHistoryNameQtrJSON} from "main/services/courseSearches";
+import fetch from "isomorphic-unfetch";
+jest.mock("isomorphic-unfetch");
+
+describe("courseSearches tests",  () => {
+  test("fetchBasicCourseJSON", async () => {
+    
+    const sampleReturnValue = {
+        "sampleKey": "sampleValue"
+    };
+
+    fetch.mockResolvedValue({
+        status: 200,
+        json: () => {
+          return sampleReturnValue;
+        },
+      });
+
+    const expectedFields = {
+        quarter: "20204",
+        department: "MATH",
+        level: "G"
+    };
+
+    const result = fetchBasicCourseJSON({},expectedFields);
+    expect(await result).toBe(sampleReturnValue);
+
+  });
+
+
+  test("fetchBasicCourseHistoryJSON", async () => {
+    
+    const sampleReturnValue = {
+        "sampleKey": "sampleValue"
+    };
+
+    fetch.mockResolvedValue({
+        status: 200,
+        json: () => {
+          return sampleReturnValue;
+        },
+      });
+
+    const expectedFields = {
+        quarter: "20204",
+        department: "MATH",
+        level: "G"
+    };
+
+    const result = await fetchBasicCourseHistoryJSON({},expectedFields);
+    expect(result).toBe(sampleReturnValue);
+
+  });
+
+  test("fetchCourseHistoryNameQtrJSON", async () => {
+    
+    const sampleReturnValue = {
+        "quarter": "20204"
+    };
+
+    fetch.mockImplementation( async () => {
+      return {
+        json : () => {
+          return sampleReturnValue;
+        }
+      };
+    }
+    );
+
+    const expectedFields = {
+      startQuarter: "20204",
+      endQuarter: "20204",
+      subjectArea: "CMPSC   ",
+      courseNumber: "130",
+      courseSuf: "A "
+    };
+
+    const result = await fetchCourseHistoryNameQtrJSON({},expectedFields);
+    expect(fetch).toHaveBeenCalledWith(`/api/public/history/coursesearch?startQtr=${expectedFields.startQuarter}&endQtr=${expectedFields.endQuarter}&subjectArea=${expectedFields.subject}&courseNumber=${expectedFields.courseNumber}&courseSuf=${expectedFields.courseSuf}`)
+    expect(result).toBe(sampleReturnValue);
+
+  });
+
+  test("fetchGeQtrJSON", async () => {
+    const sampleReturnValue = {
+      "sampleKey": "sampleValue"
+    };
+
+    fetch.mockResolvedValue({
+      status: 200,
+      json: () => {
+        return sampleReturnValue;
+      },
+    });
```

Comment:
No... we use useBackend now, and there is a whole other way of mocking with axios.

Please follow the standards of the new code base, not the old one.  That's one of the whole reasons we are rewriting the app from scratch,so that there will be a single set of dependencies and techniques across the code base.